__tests__/no-promise-in-callback.js

```diff
@@ -20,7 +20,7 @@ ruleTester.run('no-promise-in-callback', rule, {
     // arrow functions and other things
     'go(() => Promise.resolve(4))',
     'go((errrr) => a.then(b))',
```

Comment:
Not sure why the testing was unhappy when I tried changing this to `err/error/errr`, but I reverted it
```
    Message:
      Should have no errors but had 1: [
      {
        ruleId: 'no-promise-in-callback',
        severity: 1,
        message: 'Avoid using promises inside of callbacks.',
        line: 1,
        column: 13,
        nodeType: 'MemberExpression',
        endLine: 1,
        endColumn: 19
      }
    ]
```